tests/flows/test_vesting.py

```diff
@@ -6,54 +6,30 @@
 from web3 import exceptions
```

Comment:
In this file, every test function up to (but not including) `test_pool_creation_fails_for_incorrect_vesting_period()` doesn't have anything to do with testing vesting, and should be moved to the appropriate files. Either in this PR or in a quick follow-up one.

Reply:
I have created a separate file `test_bpool.py` and put `test_pool_creation_fails_for_incorrect_vesting_period()` there.